r/R/utilities_development.R

```diff
@@ -0,0 +1,11 @@
+#' Loads Experiment data
+#'
+#' @param exp_id
+#'
+#' @return
+#' @export
+#'
+#' @examples
+load_experiment_data <- function(exp_id = "test") {
+  return(readRDS(paste0("../data/", exp_id, "/r.rds", sep = "")))
```

Comment:
instead of having to manually insert `/` after each folder name, I recommend that you change `sep` to be `/`

Reply:
This file should be deleted. This was an old  set of development functions that I used locally and shared in the repo for other bioinformaticians.